lib/logic/child_tasks_cubit.dart

```diff
@@ -7,21 +7,25 @@ import 'package:fokus/model/ui/task/ui_task_instance.dart';
 import 'package:fokus/services/data/data_repository.dart';
 import 'package:fokus/services/plan_repeatability_service.dart';
 import 'package:fokus/services/task_instance_service.dart';
+import 'package:fokus/services/task_keeper_service.dart';
 import 'package:fokus/utils/duration_utils.dart';
 import 'package:get_it/get_it.dart';
 import 'package:mongo_dart/mongo_dart.dart';
 
 class ChildTasksCubit extends ReloadableCubit {
```

Comment:
Why is it called that anyway? It loads Plan Instance and has nothing to do with child so maybe lets call it PlanInstanceCubit?

Reply:
We should rename almost all cubits - that's just the way we named cubits earlier :)

Reply:
Not entirely: most cubits compile screen-specific data and depend on the current user and role. This does not, it simply loads plan instance by ID and therefore is pretty generic